src/config.py

```diff
@@ -45,4 +45,4 @@ def _convert_to_bool(x: str) -> bool:
     os.getenv("TARGET_LANGUAGES", "en").lower().split(",")
 )  # comma-separated 2-letter ISO codes
 ENCODER_SUPPORTED_LANGUAGES: Set[str] = {"en"}
-FILES_TO_PARSE: str = os.getenv("files_to_parse")
```

Comment:
Below is the definition of env vars in the pulumi job flow. Therefore, I believe the original os.getenv("files_to_parse") would be correct here?

"ContainerOverrides": {
                                        "Environment": [
                                            {
                                                "Name": "files_to_parse",
                                                "Value.$": "$.ObjectDetails.tasks_string",
                                            },
                                        ]
                                    },

Reply:
are you sure you're not looking at the parser stage?

Reply:
we should change the name of this env var to avoid confusion

Reply:
Ahh yep you're right it caps!